cirq-rigetti/requirements.txt

```
@@ -9,6 +9,9 @@ pydantic~=1.8.2
 pyjwt~=1.7.1
 pyquil~=2.28.2; python_version < "3.7"
 pyquil~=3.0.0; python_version >= "3.7"
+# Dependencies not pinned here in pyquil pip package.
+python-rapidjson==1.0
```

```suggestion
python-rapidjson<=1.6
```

This problem is due to the latest release of rapid-json. We had a similar issue in cirq-superstaq: https://github.com/SupertechLabs/cirq-superstaq/pull/289